lib/test/apyfixedarray/test_methods.py

```diff
@@ -121,3 +121,134 @@ def test_round_trip_conversion():
     for bit_pattern in range(256):
         a = APyFixedArray([bit_pattern], int_bits=4, frac_bits=4)
         assert (APyFixedArray.from_float(a.to_numpy(), 4, 4)).is_identical(a)
+
+
+def test_reshape():
+    nums = [
+        5,
+        10,
+        15,
+        20,
+        25,
+        30,
+        35,
+        40,
+        45,
+        50,
+        55,
+        60,
+    ]
+
+    # Creating 1D array to be reshaped
+    arr = APyFixedArray(nums, bits=20, int_bits=10)
+
+    possible_shapes = [(1, 12), (2, 6), (3, 4), (4, 3), (6, 2), (12, 1)]
+    possible_shapes += [(-1, 12), (2, -1), (3, -1), (4, -1), (-1, 3), (-1, 2), (12, -1)]
+
+    for shape in possible_shapes:
```

Comment:
It is possible to use `pytest.parametrize` https://docs.pytest.org/en/7.0.x/how-to/parametrize.html to "loop over tests". Hard to say which is "best", but a benefit of parametrization is that all tests are executed, now it will die after the first failure and one cannot tell if the remaining are OK or not.